Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -653,7 +670,7 @@ public SelectGestureRecognizer() : base(Tapped)
 
 					var cell = table.CellAt(_lastPath) as ContextActionsCell;
 
-					return cell != null;
+					return cell != null && (!cell.IsOpen || !cell.IsTouchInDisabledButton(pos));
```

Comment:
@jassmith I'm checking for `IsOpen` to make sure context actions are visible and we are not inadvertently hit testing actual cell content. In my testing, tapping on the left part of the cell closes the menu instead of selecting the cell. If you look at iOS Mail app, it seems to behave the same with each email row.